charts/prow/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: '{{ template "fullname" . }}'
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "prow-ingress"
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "gce"
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  tls:
+  - hosts:
+    - prow.cnct.io
```

Comment:
yes is tricky because we need a dns specified name for a domain that we control.  Do you have control over the k8s.io domain?  Let me update the README to say something about this and change the default value.  This value is what gets used in the created webhooks.

We want this to be usable by anyone so let me rework it to be more flexible.